core/kazoo_number_manager/src/carriers/knm_managed.erl

```diff
@@ -138,8 +137,9 @@ acquire_number(Number) ->
 disconnect_number(Number) ->
     lager:debug("disconnecting number ~s"
                ,[knm_phone_number:number(knm_number:phone_number(Number))]),
-    update_doc(Number, [{?PVT_STATE, ?NUMBER_STATE_RELEASED}
-                       ,{?PVT_ASSIGNED_TO, <<>>}
+    update_doc(Number, [{?PVT_STATE, ?NUMBER_STATE_AVAILABLE}
+                       ,{?PVT_ASSIGNED_TO, null}
```

Comment:
I wonder if that null field can get in couch (if that field is not there yet) and cancel out a future non-null setting of that same field?